web/packages/teleterm/src/ui/Vnet/DocumentVnetInfo.tsx

```diff
@@ -0,0 +1,295 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useMemo } from 'react';
+import styled from 'styled-components';
+
+import { Box, Button, Flex, H1, Stack } from 'design';
+import { NewTab } from 'design/Icon';
+import { H2, H3, P1, P2 } from 'design/Text/Text';
+import { DemoTerminal } from 'shared/components/DemoTerminal';
+
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import Document from 'teleterm/ui/Document';
+import { useWorkspaceContext } from 'teleterm/ui/Documents';
+import type * as docTypes from 'teleterm/ui/services/workspacesService';
+import { useConnectionsContext } from 'teleterm/ui/TopBar/Connections/connectionsContext';
+import { routing } from 'teleterm/ui/uri';
+
+import appAccessPng from './app-access.png';
+import { useVnetContext } from './vnetContext';
+
+export function DocumentVnetInfo(props: {
+  visible: boolean;
+  doc: docTypes.DocumentVnetInfo;
+}) {
+  const { mainProcessClient } = useAppContext();
+  const {
+    start,
+    startAttempt,
+    stop: stopVnet,
+    stopAttempt,
+    status,
+  } = useVnetContext();
+  const { open: openConnectionsPanel } = useConnectionsContext();
+  const userAtHost = useMemo(() => {
+    const { hostname, username } = mainProcessClient.getRuntimeSettings();
+    return `${username}@${hostname}`;
+  }, [mainProcessClient]);
+  const { rootClusterUri } = useWorkspaceContext();
+  const proxyHostname = routing.parseClusterName(rootClusterUri);
+
+  const startVnet = () => {
+    openConnectionsPanel('vnet');
+    void start();
+  };
+
+  return (
+    <Document visible={props.visible}>
+      <Stack
+        gap={6}
+        maxWidth="1400px"
+        width="100%"
+        mx="auto"
+        p={{ _: 6, 900: 9, 1200: 11 }}
```

Comment:
i've never seen this usage before. does `_: 6` mean default breakpoint?

Reply:
Yeah, `_` means "no breakpoint". https://github.com/styled-system/styled-system/blob/v5.1.5/docs/responsive-styles.md#using-objects

Breakpoints in Chakra work in a similar way. https://www.chakra-ui.com/docs/theming/customization/breakpoints

I think we haven't used it before because we've been on styled-system 3.1 for so long and this was added only in 4.0. [styled-system <4.0 had a much worse solution for breakpoints](https://github.com/styled-system/styled-system/blob/v3.1.3/docs/responsive-styles.md), so it's no wonder we weren't using it.